infra.bs

```diff
@@ -280,6 +280,81 @@ quotes and monospace font.
 <a>ASCII lowercase</a> of <var>B</var>.
 <!-- TODO: define string equals? -->
 
+To <dfn data-export="">collect a sequence of characters</dfn> means that the following algorithm
+must be run, with <var>characters</var> being the set of characters that can be collected:</p>
+
+<ol>
+
```

Comment:
This newline and the one before `</ol>`  can be removed. Same for the later algorithm.